compiler/optimizer/RematTools.cpp

```diff
@@ -242,42 +303,80 @@ void RematTools::walkNodesCalculatingRematSafety(TR::Compilation *comp,
       traceMsg(comp, "\n");
       }
    }
-   
+
 /*
  * This method walks the nodes between the start treetop inclusive and
  * the end treetop exclusive. When it encounters a node in the scanTargets
  * it adds the symref of the scan target to its set of interest and continues
  * scanning. Any symrefs killed after they have been added to the set of interest
  * are added to the unsafeSymRefs vector for later use
+ * Returns true to indicate it successfully completed the analysis.
```

Comment:
Please add a comment describing what should be in blocksToVisit for a single block walk and a multi-block walk.